### ISSUE TITLE
Save model into proper directory structure

### ISSUE THREAD

**Issue by TenzinCHW** [TenzinCHW/Algorithmic-Game-Theory#4]
Directory structure should be
/results
    /{model_type}
        /{run_name}
            /models
            /images
            /bench

There's also a /results/ref directory which will store the ground truth results.
{model_type} differentiates between major differences in model i.e. different loss function. {run_name} differentiates between hyperparameters.

**Comment by TenzinCHW** [comment]
Benchmark results will be saved to a file called 'fid.json' and 'inception.json'.
